utils/docker-ce/files/dockerd.init

```diff
@@ -145,6 +145,7 @@ process_config() {
 	json_add_array "hosts"
 	config_list_foreach globals hosts json_add_array_string
 	json_close_array
+	json_add_boolean iptables 0
```

Comment:
I think for this to go in we'd have to make it configurable and only add it if it's `false` as per another comment